lib/io/iob.h

```diff
@@ -175,6 +175,42 @@ extern "C"{
      */
     int write_angle_offset(int id, double offset);
 
+    /**
+     * @brief read lower joint limit[rad]
+     * @param id	joint id
+     * @param angle	lower joint limit[rad]
+     * @retval          TRUE this function is supported
+     * @retval          FALSE otherwise
+     */
+    int read_llimit_angle(int id, double *angle);
+
+    /**
+     * @brief read upper joint limit[rad]
+     * @param id	joint id
+     * @param angle	upper joint limit[rad]
+     * @retval          TRUE this function is supported
+     * @retval          FALSE otherwise
+     */
+    int read_ulimit_angle(int id, double *angle);
+
+    /**
+     * @brief write lower joint limit[rad]
+     * @param id	joint id
+     * @param angle	lower joint limit[rad]
+     * @retval          TRUE this function is supported
+     * @retval          FALSE otherwise
+     */
+    int write_llimit_angle(int id, double angle);
+
+    /**
+     * @brief write upper joint limit[rad]
+     * @param id	joint id
+     * @param angle	upper joint limit[rad]
+     * @retval          TRUE this function is supported
+     * @retval          FALSE otherwise
+     */
+    int write_ulimit_angle(int id, double angle);
+    
```

Comment:
Are these necessary? They are not called in this PR.

Reply:
@fkanehiro In this PR they are not used, but in the `PosTorqueCtrl-rebase` branch of `hrp5p-iob`, that I will try to merge soon, I have a standalone program called `modify_joint_limits` (https://github.com/isri-aist/hrp5p-iob/blob/PosTorqueCtrl-rebase/iob/modify_joint_limits.cpp) that uses those functions

Reply:
If they are not relevant to this PR, I think they should be removed from this PR.

In addition, as adding APIs will break compatibility, you need to use `ROBOT_IOB_VERSION` to add APIs.
Moreover, I prefer not to add these APIs. Because they will enable normal users to implement dangerous programs.

Reply:
Got it... I have removed these APIs in https://github.com/fkanehiro/hrpsys-base/pull/1326/commits/d8d245f32291e446f4ea60dc42accaec5f84826a